integration-tests/emulated/Cargo.toml

```diff
@@ -78,3 +84,11 @@ runtime-benchmarks = [
   "runtime-common/runtime-benchmarks",
   "spiritnet-runtime/runtime-benchmarks",
 ]
+
+try-runtime = [
+  "ctype/try-runtime",
+  "did/try-runtime",
+  "pallet-bags-list/try-runtime",
+  "peregrine-runtime/try-runtime",
+  "spiritnet-runtime/try-runtime",
+]
```

Comment:
Do we need this?

Reply:
Yes. I tested it out. Otherwise, the crate will not compile